dockertest/output/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+API compatibility: on 2016-04-29 output.py got split up into a
+package with four smaller, more maintainable submodules. This init
+allows existing clients to continue importing as a single unit.
+"""
+
+# ARGH. Without this, 'import xceptions' in a submodule fails because
+# dockertest isn't in the search path.
+from os.path import dirname
+__path__ += [dirname(__path__[0])]
+
+# pylint: disable=W0401
+from . dockertime import *
+from . dockerversion import *
+from . texttable import *
+from . validate import *
```

Comment:
This is bad-juju, can't ``import *`` b/c it pollutes the namespace.  e.g. ``validate`` may define something with a name that collides with ``dockertime``, however the collision is "out of it's jurisdiction" as it were.

At least until the next minor version,  you'll have to list out the names specificity that you want to import from each module, into the package.